File: app/src/main/java/com/example/app/jasper/routinedeveloper_v2/model/Todo.java
package com.example.app.jasper.routinedeveloper_v2.model;

import java.io.Serializable;

public class Todo implements Serializable {

    long id =-1;
    String name;
    boolean done;

    public Todo(long id, String name, boolean done) {
        this.id = id;
        this.name = name;
        this.done = done;
    }

    public Todo(String name){
        this.name = name;
    }

    public Todo() {
    }

    public long getId() {
        return id;
    }
    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }

    public boolean isDone() {
        return this.done;
    }
    public void setDone(boolean done) {
        this.done = done;
    }

    public String toString(){
        return this.name;
    }
}

File: app/src/main/java/com/example/app/jasper/routinedeveloper_v2/model/Todo.kt
package com.example.app.jasper.routinedeveloper_v2.model

import java.io.Serializable

class Todo : Serializable {
    var id: Long = -1
    var name: String? = null
    var isChecked = false
    var position = 0
    var doneCounts = 0
    var undoneCounts = 0

    override fun toString(): String {
        return "id: $ id, name: $name, is done: $isChecked, position: $position, doneCounts: $doneCounts, undoneCounts: $undoneCounts"
    }
}